voicevox/audio_query.py

```diff
@@ -13,17 +13,17 @@ class Mora:
     Attributes
     ----------
     text: str
-        文字
+        Text to synthesize
```

Comment:
Mora are returned from `/accent_phrases`, thus they are already parsed - the text is not parsed again because it is already converted into the other data which the synthesizer actually uses